lib/ipaddr.rb

```diff
@@ -111,7 +111,11 @@ def self.new_ntoh(addr)
   # Convert a network byte ordered string form of an IP address into
   # human readable form.
```

Comment:
Could you add something like this here?  "It expects a string encoded in Encoding::ASCII_8BIT."